src/components/LinkedLicenseCard/LinkedLicenseCard.test.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { KeyValue } from '@folio/stripes-testing';
+import { MemoryRouter } from 'react-router-dom';
+import translationsProperties from '../../../test/helpers';
+import LinkedLicenseCard from './LinkedLicenseCard';
+import { futureLicense, historicalLicense } from './testResources';
+
+jest.mock('@folio/stripes-erm-components', () => ({
+  ...jest.requireActual('@folio/stripes-erm-components'),
+  LicenseCard: () => <div>LicenseCard</div>,
+}));
+jest.mock('../LicenseAmendmentList', () => () => <div>LicenseAmendmentList</div>);
+
+let renderComponent;
+describe('LinkedLicenseCard', () => {
+    describe('Future license card', () => {
+      beforeEach(() => {
+        renderComponent = renderWithIntl(
+          <MemoryRouter>
+            <LinkedLicenseCard
+              id="agreement-future-license-0"
+              license={futureLicense}
+            />
+          </MemoryRouter>,
+          translationsProperties
+        );
+      });
+
+      test('renders LinkedLicenseCard component', () => {
+        const { getByTestId } = renderComponent;
+        expect(getByTestId('linkedLicense')).toBeInTheDocument();
+       });
+
+       test('renders a link with the license name', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('link', { name: futureLicense.remoteId_object.name })).toBeInTheDocument();
```

Comment:
Lets just test the actual expected value instead of the accessing it vai the properties of the `futureLicense`. That way it will be easy when reading through these tests. Please change them across this file